docs/conceptual/cryptography/session-keys.md

```diff
@@ -28,10 +37,21 @@ mod app {
 }
 ```
 
-> **Note:** This code is just an example of the Substrate node at the [time of writing](https://github.com/paritytech/substrate/tree/9fa8589d9b8cfe8716e9e4c48f9e3f238c1e502f). Refer to the runtime for the most up-to-date implementation.
+> **Note:** This code is just an example of the Substrate node at the
```

Comment:
```suggestion
> This code is just an example of the Substrate node at the
```

My stylistic preference. It wouldn't be written in that yellow box if we didn't want readers to note it.

Reply:
Up to you though, Joe. I'm fine either way.

Reply:
I love cutting extra words!